### PR TITLE
Merge main into next-release/main

### PR DESCRIPTION
**Pull request by timngyn**
#### Description of changes:
Includes these PRs:
- https://github.com/aws-amplify/docs/pull/7346
- https://github.com/aws-amplify/docs/pull/6471
- https://github.com/aws-amplify/docs/pull/7413
- https://github.com/aws-amplify/docs/pull/7408

### Instructions

**If this PR should not be merged upon approval for any reason, please submit as a DRAFT**

Which product(s) are affected by this PR (if applicable)?
- [ ] amplify-cli
- [ ] amplify-ui
- [ ] amplify-studio
- [ ] amplify-hosting
- [ ] amplify-libraries

Which platform(s) are affected by this PR (if applicable)?
- [ ] JS
- [ ] Swift
- [ ] Android
- [ ] Flutter
- [ ] React Native

**Please add the product(s)/platform(s) affected to the PR title**

#### Checks

- [ ] Does this PR conform to [the styleguide](https://github.com/aws-amplify/docs/blob/main/STYLEGUIDE.md)?

- [ ] Does this PR include filetypes other than markdown or images? Please add or update unit tests accordingly.

- [ ] Are any files being deleted with this PR? If so, have the needed redirects been created?

- [ ] Are all links in MDX files using the MDX link syntax rather than HTML link syntax? <br /> 
      _ref: MDX: `[link](https://docs.amplify.aws/)` 
            HTML: `<a href="https://docs.amplify.aws/">link</a>`_

### When this PR is ready to merge, please check the box below
- [ ] Ready to merge

_By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license._

### CODE REVIEW

File: src/pages/[platform]/build-a-backend/server-side-rendering/nextjs-app-router-server-components/index.mdx
Comment (timngyn) [inline src/pages/[platform]/build-a-backend/server-side-rendering/nextjs-app-router-server-components/index.mdx:1]: Hmm trying to see if this file should exist

**Review by reesscot — DISMISSED** [review]

File: src/pages/[platform]/build-a-backend/data/set-up-data/index.mdx [hunk 1]
```diff
@@ -113,11 +113,11 @@ npx ampx sandbox --outputs-format dart --outputs-out-dir lib
 
 ## Connect your application code to the data backend
 
-Once the cloud sandbox is up and running, it will also create an `amplifyconfiguration.json` file, which includes the relevant connection information to your data backend, like your API endpoint URL and API key.
+Once the cloud sandbox is up and running, it will also create an `amplify_outputs.json` file, which includes the relevant connection information to your data backend, like your API endpoint URL and API key.
 
 To connect your frontend code to your backend, you need to:
 
-1. configure the Amplify library with the Amplify client configuration file
+1. configure the Amplify library with the Amplify client configuration file (`amplify_outputs.json`)
```
Comment (katiegoines) [inline src/pages/[platform]/build-a-backend/data/set-up-data/index.mdx:120]: It looks like elsewhere “configuration file” is replaced with “output file”. Should it be here as well?

**Review by reesscot — DISMISSED** [review]

File: src/pages/[platform]/reference/cli-commands/index.mdx [hunk 2]
```diff
@@ -225,7 +225,7 @@ npx ampx generate outputs --stack amplify-nextamplifygen2-josef-sandbox-ca85e108
 
 #### Use with mobile applications
 
-Similar to `sandbox`, you can specify an alternate configuration file format by using `--format`:
+Similar to `sandbox`, you can specify an alternate outputs file format by using `--format`:
```
Comment (timngyn) [inline src/pages/[platform]/reference/cli-commands/index.mdx:228]: This file only had this change from this PR: https://github.com/aws-amplify/docs/pull/7346/files#diff-61d6c0f99c1aa1d1c1e497cfcfc20a3ae905ccd606de326c7f416eeacf467977

The other lines were removed from `next-release/main`, but not `main`

File: src/pages/[platform]/build-a-backend/data/set-up-data/index.mdx (hunk 1 above)
Comment (timngyn) [reply src/pages/[platform]/build-a-backend/data/set-up-data/index.mdx:120]: it probably should, but I didn't want to add things manually for this merge

**Review by katiegoines — DISMISSED** [review]

**Review by hbuchel — APPROVED** [review]

**Pull request merged**